tests/darcy_outflow.prm

```diff
@@ -0,0 +1,190 @@
+  end
+end
+
+subsection Boundary composition model
+  set Fixed composition boundary indicators = bottom, top
+  set Allow fixed composition on outflow boundaries = false
+  set List of model names = initial composition
+end
+
+subsection Material model
+  set Model name                                        = reactive fluid transport
+  set Material averaging                                = geometric average only viscosity
+
+  subsection Reactive Fluid Transport Model
+    set Base model                                       = visco plastic
+    set Reference fluid density                          = 1000 # density of water
+    set Shear to bulk viscosity ratio                    = 1
+    set Reference fluid viscosity                        = 10
+    set Reference permeability                           = 1e-6
+    set Exponential fluid weakening factor               = 0
+    set Fluid compressibility                            = 0
+    set Fluid reaction time scale for operator splitting = 1e50
+    set Fluid-solid reaction scheme                      = zero solubility
+  end
+
+  subsection Visco Plastic
+    set Viscosity averaging scheme                      = geometric
+    set Viscous flow law                                = diffusion
+    set Prefactors for diffusion creep                  = 5e-21
+    set Stress exponents for diffusion creep            = 1.0
+    set Activation energies for diffusion creep         = 0
+    set Activation volumes for diffusion creep          = 0
+    set Densities                                       = 3000
+
+    set Angles of internal friction                     = 0
+    set Cohesions                                       = 1e50
+
+    set Minimum viscosity                               = 1e21
+    set Maximum viscosity                               = 1e21
+    set Thermal expansivities                           = 0
+  end
+end
+
+# Set the global refinement to 0, 1 km x 1 km mesh.
+subsection Mesh refinement
+  set Coarsening fraction                      = 0.1
+  set Refinement fraction                      = 0.9
+  set Initial adaptive refinement              = 2
+  set Initial global refinement                = 0
+  set Strategy                                 = composition threshold, minimum refinement function
+  set Time steps between mesh refinement       = 1
+
+  # Minimum of 4 global refinements
+  subsection Minimum refinement function
+    set Function expression = 0
+  end
+
+  # Refine where the porosity is bigger than 1e-6. Other compositions
+  # are set to 1e50 to ensure that we do not refine based on these
+  # compositions.
+  subsection Composition threshold
+    set Compositional field thresholds = 1e-3, 1e50, 1e50
+  end
+end
+
+
+# Output the darcy velocity
+subsection Postprocess
+  set List of postprocessors          = velocity statistics, visualization
```

Comment:
can you add the composition statistics postprocessor? Just because the value of the compositional fields is the thing we'd expect to change if things ever stopped working correctly.